newIDE/app/scripts/extract-extensions-document.js

```diff
@@ -122,6 +122,7 @@ GDevelop is built in a flexible way. In addition to [[gdevelop5:all-features|cor
         '\n' +
         convertMarkdownToDokuWikiMarkdown(extension.description) +
         '\n' +
+        (helpPageUrl ? `\n[[${helpPageUrl}|Read more...]]\n` : ``) +
```

Comment:
Could even have been in the top of the page as a "Read the detailed page about this extension".

Reply:
I don't mind either. I supposed that users will focus on the description and when reaching the end might want to continue learning more about the extension by following the link.